Week 2 Assignment 1/Factorial_W2A1_3.java

```diff
@@ -0,0 +1,15 @@
+public class Factorial_W2A1_3 {
+    static int product = 1;
+    public static void main(String args[]) {
+        int num = 4;
+        System.out.println(recursion(num));
+    }
+
+    static int recursion(int num) {
+        if(num > 0) {
```

Comment:
Consider using the terminating condition to add a terminating return type for the function. It will remove the necessity for the product variable.